doc/config.rst

```diff
@@ -1624,6 +1624,46 @@ These options influence LaTeX output.
         .. versionadded:: 1.4
      ``'preamble'``
         Additional preamble content, default empty.
+
+        This can be the location
+        for customization via ``\renewcommand``, ``\renewenvironment``,
+        ``\definecolor`` or ``\setlength`` of some defaults.
+
+        - text styling commands (they have one argument): ``\sphinx<foo>`` with
+          ``<foo>`` begin one of ``strong``, ``bfcode``, ``email``,
+          ``tablecontinued``, ``titleref``, ``menuselection``, ``accelerator``,
+          ``crossref``, ``termref``, ``optional``. By default and for backwards
+          compatibility the ``\sphinx<foo>`` expands to ``\<foo>`` hence the
+          user can choose to customize rather the latter.
+        - parameters for paragraph level environments: with ``<foo>`` one of
+          ``warning``, ``caution``, ``attention``, ``danger``, the colours
```

Comment:
rather than ``` `` ``` one should use here auto-generated links to discussion in the doc of these admonitions.